mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -62,6 +63,21 @@
 event_id = {'Sync On': 5}
 
 
+def test_orig_units():
+    """Test exposure of original channel units."""
+    with pytest.warns(RuntimeWarning, match='will be dropped'):
+        raw = read_raw_brainvision(vhdr_path)
```

Comment:
how much does the test takes to run? maybe we should start thinking on using a fixture to load the data and do it only once for the entire testsuit, and save some CI cycles.

Reply:
it's pretty fast:
```
$ pytest mne/io/brainvision/tests/test_brainvision.py::test_orig_units --durations=0
Test session starts (platform: linux, Python 3.6.6, pytest 3.7.3, pytest-sugar 0.9.1)

 mne/io/brainvision/tests/test_brainvision.py ✓                                                                 100% ██████████
=================================================== slowest test durations ====================================================
0.01s call     mne/io/brainvision/tests/test_brainvision.py::test_orig_units
0.00s setup    mne/io/brainvision/tests/test_brainvision.py::test_orig_units
0.00s teardown mne/io/brainvision/tests/test_brainvision.py::test_orig_units

Results (0.20s):
       1 passed


```